articles/azure-government-developer-guide.md

```diff
@@ -106,7 +106,7 @@ Use the following table to guide you when mapping public Microsoft Azure and SQL
 | SqlDatabaseDnsSuffix | .database.usgovcloudapi.net |
 | StorageEndpointSuffix | core.usgovcloudapi.net |
 | ActiveDirectoryAuthority | https://login-us.microsoftonline.com/ |
-| GraphUrl | https://graph.windows.net/ 
+| GraphUrl | https://graph.windows.net/
```

Comment:
Looks like the original was also missing a pipe character at the end of line 109.